api-server/common/models/user.js

```diff
@@ -376,6 +381,7 @@ export default function(User) {
   });
 
   User.about = function about(username, cb) {
+    username = username.toLowerCase();
```

Comment:
Operate on the property only after the validation check in the next line. This would throw an exception if the property is not valid.

Reply:
@raisedadead moved down again